client/__tests__/e2e/e2e.test.js

```diff
@@ -521,6 +522,17 @@ test("lasso moves after pan", async () => {
   expect(panCount).toBe(initialCount);
 });
 
+describe("auth buttons", () => {
+  test("login then logout", async () => {
```

Comment:
This is more a general question, I was looking at the authButton component and don't see any unit tests... and then I saw that we don't have tests for any components. Am I looking in the wrong place? We should add a case for displaying the `logged in as` section (but that doesn't need to be a e2e layer test)

Reply:
Yeah we don't have React component related unit tests yet. But we should start adding them! 😄